### PR TITLE
docs: fix modSecurity typo

### PR DESCRIPTION
**Pull request by michael12312**
## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Comment by efd6** [comment]
This will need to have changes in the changelog.yml and the manifest.yml.

**Comment by michael12312** [comment]
@efd6 
hi, I updated changelog and manifest, please help to review, thanks!

File: packages/modsecurity/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.1.3"
+  changes:
+    - description: Fix typo in the build/docs/README.md
+      type: doc
```
Comment (efd6) [inline packages/modsecurity/changelog.yml:5]:
```suggestion
      type: bugfix
```

Comment (michael12312) [reply packages/modsecurity/changelog.yml]: applied

**Comment by michael12312** [comment]
@efd6 
all tests passed, can you merge the PR now? 
Or if more changes needed, please let me know.

**Review by efd6 — APPROVED** [review]
Thanks

**Pull request merged**